README.md

```diff
@@ -257,3 +299,20 @@ Destination is relative to the output policy set (`out/masterfiles`).
 - Enable `services_autorun_bundles` class in `def.json`.
 - Merge any `def.json` recursively into `out/masterfiles/def.json`.
 - Copy any other files with their existing directory structure to destination.
+
+#### `bundles <bundles ...>`
+- Ensure bundles are evaluated by adding them to the bundle sequence, using `def.json`.
+- Only manipulates the bundle sequence, to ensure policy files are copied and parsed, use other build steps, for example `copy` and `policy_files`.
+
+#### `policy_files <paths ...>`
+- Add policy (`.cf`) files to `inputs` key in `def.json`, ensuring they are parsed.
```

Comment:
So, the inputs key, is also an MPFism. It ultimately populates `default:def.augments_inputs` which then must be included in inputs by the actual policy.

Initially when augments parsing moved to C from policy the inputs key would inject directly to inputs, so that applied for any policy entry that loaded the augments file. We saw some issues with duplicate definition of bundle and we backed off since direct injection to inputs provided no way to vary inputs per entry (e.g. different policy files added to inputs for `promises.cf` vs `update.cf`).

So, currently `promises.cf` uses `default:def.augments_inputs` to extend inputs but update.cf looks for `default:def.update_inputs`, which must be specified in augments via `vars` or `variables`.

Anyway, I think the wording here is a bit strong given how it actually works and how much of it is specific to MPF.

Reply:
Added clarification.